nixos/modules/services/continuous-integration/buildkite-agent.nix

```diff
@@ -74,13 +74,12 @@ in
         '';
       };
 
-      meta-data = mkOption {
-        type = types.str;
-        default = "";
-        example = "queue=default,docker=true,ruby2=true";
+      tags = mkOption {
```

Comment:
If we're dropping support for 2.x, we should also make the `buildkite-agent` to be 3.x version. Right now it is still 2.x so by default you'd get something that is not working: version 2 wants `meta-data` and not `tags`.